### PR TITLE
Increment R package version, some goodpractice checks

### PR DESCRIPTION
**Pull request by jsta**
See Discussion in #111

### CODE REVIEW

**Comment by jsta** [comment]
Also see #113

File: R/run_glm.R [hunk 1]
```diff
@@ -106,17 +111,20 @@ run_glmOSx <- function(sim_folder, verbose = TRUE, args){
   })
 }
 
-run_glmNIX = function(sim_folder, verbose=TRUE, args){
+run_glmNIX <- function(sim_folder, verbose=TRUE, args){
```
Comment (lawinslow) [inline R/run_glm.R]: Does that make you feel better?

Comment (lawinslow) [inline R/run_glm.R]: https://www.youtube.com/watch?v=SsoOG6ZeyUI

Comment (jsta) [inline R/run_glm.R]: lol, its a curse

**Pull request merged**